packages/react/src/tokens/themes/theme.types.ts

```diff
@@ -46,6 +46,8 @@ export interface Theme {
 }
 
 export type Colors = {
+  white?: string;
```

Comment:
I think black and white make more sense to be in `neutral` to be concistent.

Reply:
I put it back to the "brand colors" since it's too much effort to only include it on the other color types because of the current strong typing